ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-master.yaml

```
@@ -92,6 +92,10 @@ tests:
   commands: make gosec
   container:
     from: src
+- as: lint
+  commands: make GOLANGCI_LINT_CACHE=/tmp/golangci-cache make test/lint
```

I don't think we need this make command at the start 

```suggestion
  commands:  GOLANGCI_LINT_CACHE=/tmp/golangci-cache make test/lint
```

---

Thanks for spotting it! Updated